Django/BookLibrary/library/forms.py

```diff
@@ -0,0 +1,8 @@
+from django import forms
+
+#Simple form to get the Book name from user
+class SearchBook(forms.Form):
+    title = forms.CharField(label='Search a book', 
```

Comment:
This we discussed already in class. But try to avoid forms as it is not used much anymore. Use models.Model always